tests/test_reporters.py

```diff
@@ -810,3 +810,41 @@ def test_delta_reporter__single_file_coverage_changed():
             "Cover": "+16.67%"
         }
     }
+
+
+def test_phorge_json_reporter():
+    from pycobertura.reporters import PhorgeJsonReporter
+
+    cobertura = make_cobertura('tests/dummy.source1/coverage.xml')
+    report = PhorgeJsonReporter(cobertura)
+
+    assert report.generate() == """\
+{
+    "dummy/__init__.py": "",
+    "dummy/dummy.py": "CCNCUU",
```

Comment:
What is N?

Reply:
This is described in https://we.phorge.it/book/phorge/article/arcanist_coverage/#building-coverage-support (I have put this link into README): N means "not executable"; C means "covered" and "U" means "not covered"